security/src/main/java/com/fincity/security/service/appregistration/AppRegistrationIntegrationService.java

```diff
@@ -87,7 +85,7 @@ public Mono<Integer> delete(ULong id) {
                 return FlatMapUtil.flatMapMono(
                                 () -> this.read(id),
                                 e -> super.delete(id).flatMap(this.cacheService.evictFunction(CACHE_NAME_INTEGRATION_ID,
```

Comment:
I don't think this will work, use evictFunctionWithKeyFunction and change the getCacheKeys to return string.